external/solr/src/main/java/com/digitalpebble/stormcrawler/solr/bolt/IndexerBolt.java

```diff
@@ -104,7 +104,7 @@ public void execute(Tuple tuple) {
                 // Distinguish the value used for indexing
                 // from the one used for the status
                 String normalisedurl = valueForURL(tuple);
-                doc.addField(fieldNameForURL(), normalisedurl);
+                doc.addField(fieldNameForURL(), getDocumentID(metadata, normalisedurl));
```

Comment:
This is to store the URL, not the doc ID. I don't think we should call getDocumentID here, even if it returns the normalised URL.